docs/docsite/rst/community/development_process.rst

```diff
@@ -140,7 +140,8 @@ Changelogs
 ----------
 
 Changelogs help users and developers keep up with changes to Ansible.
-Ansible builds a changelog for each release from fragments. You **must** add a changelog fragment to any PR that changes functionality or fixes a bug.
+Ansible builds a changelog for each release from fragments.
+You **must** add a changelog fragment to any PR that changes functionality or fixes a bug in ansible-base or collection.
```

Comment:
```suggestion
You **must** add a changelog fragment to any PR that changes functionality or fixes a bug in ansible-base.
```
This depends on the collection's rules. For some collection you have to, for other collection you need to use a different changelog generation mechanism, and yet other collection do not have a changelog (yet).

Reply:
makes sense to remove, done